src/path_manager.c

```diff
@@ -736,13 +738,35 @@ static void handle_mptcp_event(struct l_genl_msg *msg, void *user_data)
 static void family_appeared(struct l_genl_family_info const *info,
                             void *user_data)
 {
-        (void) info;
+        struct mptcpd_pm *const pm = user_data;
+
+        if (info == NULL) {
+                /*
+                  Request for "mptcp" generic netlink family failed.
+                  Wait for it to appear in case it is loaded into the
+                  kernel after mptcpd has started.
+                */
+
+                l_debug("Request for \""
+                        MPTCP_GENL_NAME
+                        "\" Generic Netlink family failed.  Waiting.");
+
+                if (l_genl_add_family_watch(pm->genl,
```

Comment:
On second thought, this won't work for the case where the `netlink` path manager kernel module is removed dynamically.  I'll move this call back to where it was, but after the call to `l_genl_request_family()`.

Reply:
Done.